GUI/MainInstall.cs

```diff
@@ -147,30 +147,43 @@ public async void InstallModuleDriver(IRegistryQuerier registry, CkanModule modu
                 installCanceled = true;
             };
 
-            bool resolvedAllProvidedMods = false;
+             // checks if all actions were successfull
+             bool resolvedAllProvidedMods = false;
+            // uninstall/installs/upgrades until every list is empty
+            // if the queue is NOT empty, resolvedAllProvidedMods is set to false until the action is done
             while (!resolvedAllProvidedMods)
             {
                 try
                 {
                     e.Result = new KeyValuePair<bool, ModChanges>(false, opts.Key);
-                    if (!installCanceled && toUninstall.Count > 0)
+                    if (toUninstall.Count > 0)
                     {
-                        installer.UninstallList(toUninstall);
+                        resolvedAllProvidedMods = false;
+                        if (!installCanceled) {
+                            installer.UninstallList(toUninstall);
+                            resolvedAllProvidedMods = true;
```

Comment:
This variable is specifically to track handling of TooManyModsProvideKraken. It should not be getting set after uninstalls like this.

Reply:
Ohhh, and I thought that this was a strange variable name for a simple "Just keep going until everything is done" while loop... 
Yeah so basically I create a new bool for my "everything done?" checks.